sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,89 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient.
+        They can be expressed in numerous ways, including as a product of binomial coefficients or of factorials:
+    .. math:: {n \choose k_1, k_2, \ldots, k_m} = \frac{n!}{k_1!\, k_2! \cdots k_m!} = {k_1\choose k_1}{k_1+k_2\choose k_2}\cdots{k_1+k_2+\cdots+k_m\choose k_m}
```

Comment:
So here, [this](https://en.wikipedia.org/wiki/Multinomial_theorem#Multinomial_coefficients) representation is used. What if sum of k1, k2, k3, ..., kn is not n?

Reply:
I haven't accounted for such a situation ... this is what Wikipedia says 

> The sum is taken over all combinations of nonnegative integer indices k1 through km such that the sum of all ki is n. That is, for each term in the expansion, the exponents of the xi must add up to n.

Reply:
But what if user gives the wrong input?

Reply:
We can consider it as GIGO situation or raise an error for that. I'd prefer former one as for symbolic inputs it will be difficult to test the sum of k1, k2, k3, ...kn and verify it that it matches n. Some thought is needed here.

Reply:
since multinomial takes only a tuple from the user ... the n is automatically calculated using the Add func ... so technically the user cannot make a mistake ...

Reply:
except in the case of one input ... 😅 ... Ahhh got it thanks !!!

Reply:
I'd have to add a check for input length greater than 2 😄  ... right ?

Reply:
> since multinomial takes only a tuple from the user ... the n is automatically calculated using the Add func

So you have used, (k1 + k2 + k3 + . . . + kn)!/(k1!k2!k3!...kn!) in your code?

Reply:
Yes I've done that! 
```
factorial(Add(*k))/Mul(*(factorial(x) for x in k))
```
and
```
gamma(Add(*k) + 1)/Mul(*(gamma(x + 1) for x in k))
```

Reply:
Please update the doc strings accordingly. It appears from the doc strings that (n!)/k1!k2!k3!...kn! is used. Consistency should be there.

Reply:
alright ... I've removed the first part with the n and replace n! with (k1+k2+...+km)!

Reply:
@czgdp1807 I've committed the changes i've made ... let me know what you think